TopQuarkAnalysis/TopEventProducers/interface/StGenEventReco.h

```diff
@@ -1,17 +0,0 @@
-#include "FWCore/Framework/interface/EDProducer.h"
-#include "FWCore/Framework/interface/Event.h"
-#include "FWCore/Utilities/interface/InputTag.h"
-#include "FWCore/ParameterSet/interface/ParameterSet.h"
-
-#include "AnalysisDataFormats/TopObjects/interface/StGenEvent.h"
-
-class StGenEventReco : public edm::EDProducer {
```

Comment:
I wonder why this file didn't get fully removed, even if empty

Reply:
Good catch! (I forgot `git rm`)